README.md

```diff
@@ -49,70 +79,135 @@ const client = new Realtime(ABLY_API_KEY);
+const space = spaces.get('demoSlideshow');
+
+// Register a listener to subscribe to events of when users enter or leave the space
 space.on('membersUpdate', (members) => {
   console.log(members);
 });
-```
 
-### Example payload returned
+// Enter a space, publishing a memberUpdate event, including optional user data
+space.enter({
+  username: "Claire Lemons",
+  avatar: "https://slides-internal.com/users/clemons.png",
+});
+```
 
-1. When a member has entered the space via `space.enter()`:
+The following is an example `membersUpdate` event received by subscribers when a user enters a space:
 
 ```json
 [
   {
-    "clientId": "user1234",
+    "clientId": "clemons@slides.com",
     "isConnected": true,
     "lastEvent": {
       "name": "enter",
       "timestamp": 1677595689759
     },
     "location": null,
     "profileData": {
-      "username": "Example User",
-      "avatar": "https://example.com/users/1234.png"
+      "username": "Claire Lemons",
+      "avatar": "https://slides-internal.com/users/clemons.png"
     }
   }
 ]
 ```
 
-2. When a member has left the space in one of the following ways:
+### Location updates
+
+User locations enable you to track where clients are within an application. Subscribe to all location updates, those for a specific location, such as a single UI element, or a specific client.
+
+```ts
+// Register a listener to subscribe to events of when users change location
+space.locations.on('locationUpdate', (update) => {
+  console.log(update);
+});
+
+// Publish locationUpdate event with a client's location when they select a UI element or change web pages
+space.locations.set({slide: '3', component: 'slide-title'});
+
+// Create a tracker to only publish locationUpdate events for a specific user using their clientId
+const memberTracker = space.locations.createTracker(
+  (change) => change.member.clientId === 'clemons@slides.com'
+);
 
-- called `space.leave()`
-- has closed the tab
-- abruptly disconnected from the internet for more than 2min (which is configurable via [`offlineTimeout`](https://github.com/ably-labs/spaces/blob/main/docs/class-definitions.md#offlinetimeout):
+// Register a listener to subscribe to events for a tracker
+memberTracker.on((change) => {
+  console.log(change);
+});
+
+// Create a tracker to only publish locationUpdate events for a specific location, such as a UI element or spreadsheet cell
+const locationTracker = space.locations.createTracker(
+  (change) => change.previousLocation === 'slide-title'
+);
+
+// Register a listener to subscribe to events for a tracker
+locationTracker.on((change) => {
+  console.log(change);
+});
+```
+
+The following is an example `locationUpdate` event received by subscribers when a user changes location:
 
 ```json
-[
-  {
-    "clientId": "user1234",
-    "isConnected": false,
-    "lastEvent": {
-      "name": "enter",
-      "timestamp": 1677595689759
-    },
-    "location": null,
+{
+    "member": {
+    "clientId": "clemons@slides.com",
```

Comment:
I think this more then likely should be the unique id of user of the application, emails can be non unique (multiple accounts and such)

Reply:
(also some of the indentation in this code sample has gone wrong!)

Reply:
Fair point - I've updated to a random user string instead. Also fixed indentation.